### DISCUSSION TITLE
Can I downgrade telegraf version?

### DISCUSSION THREAD

**Discussion by saycubed** [telegraf/telegraf#1517] [Q&A]
A week ago I worked on an old bot and it wasn't working locally until I upgraded the telegraf version. Now it won't work remotely on heroku!. It runs on micro-bot which I hear is not being maintained any longer. Can I revert to the old version of telegraf, how would I do that, and do you think that is the solution?

**Comment by Latvels** [comment]
Hi. You need to enter the following command in the console: npm i telegraf@3.0.0
Where after @ is the version you need.
